.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "cardano-repo-tool"]
 	path = cardano-repo-tool
-	url = https://github.com/input-output-hk/cardano-repo-tool.git
+	url = git@github.com:input-output-hk/cardano-repo-tool.git
```

Comment:
I think they want to keep the 'https://' method so anybody can download and execute it.

Reply:
Fixed.